src/RadixDlt.NetworkGateway.GatewayApi/gateway-api-schema.yaml

```diff
@@ -1960,14 +1982,19 @@ components:
       type: object
       properties:
         ancestor_identities:
+          description: (false by default) if set to true, ancestor addresses - parent_address, owner_address and global_address for entities are returned.
```

Comment:
IMO it's kinda pointless to point out that default value of boolean property in optional object is `false` cause it's quite obvious in this context. However, if we really want to be that explicit in our docs I'd say we should move this information to the very back of the description, i.e. "(false by default) bla bla" => "bla bla. Defaults to `false`."

Reply:
BTW use backticks whenever we reference a value, i.e. "false" => "`false`"